.github/workflows/tests.yml

```
@@ -0,0 +1,36 @@
+---
+name: tests
+on: [push]
+jobs:
+  plugin-tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    container:
+      image: buildkite/plugin-tester:latest
+      volumes:
+        - "${{github.workspace}}:/plugin"
+    steps:
+      - uses: actions/checkout@v2
+      - name: tests
+        run: bats tests/
+  plugin-lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    container:
+      image: buildkite/plugin-linter:latest
+      volumes:
+        - "${{github.workspace}}:/plugin"
+    steps:
+      - uses: actions/checkout@v2
+      - name: lint
+        run: lint --id envato/stop-the-line
```

The id here looks wrong.

```suggestion
        run: lint --id envato/ejson2env